packages/vue/executors.json

```diff
@@ -0,0 +1,4 @@
+{
+  "builders": {},
```

Comment:
Building on my comment about Angular CLI, do we really want "builders" here on a package that will never be used in an Angular CLI workspace?